cla_backend/libs/eligibility_calculator/tests/cfe_civil/test_property.py

```diff
@@ -0,0 +1,102 @@
+from unittest import TestCase
+
+from cla_backend.libs.eligibility_calculator.cfe_civil.property import translate_property
+
+
+class TestTranslateProperty(TestCase):
+    def test_multiple_properties(self):
+        houses = [
+            {'disputed': False,
+             'main': True,
+             'share': 80,
+             'value': 100 * 1000 * 100,
+             'mortgage_left': 50 * 1000 * 100},
```

Comment:
@starswan What do you think of using `50000 * 100` instead of `50 * 1000 * 100`. It will be helpful for pattern matching :)

Reply:
I thought this was easier to read as 50,000

Reply:
Whilst there's something to be said for ruby's `50_000` style, in Python it's normal to just go with it in full e.g. `50000` 🤷